cvat/apps/auto_annotation/inference_engine.py

```diff
@@ -24,7 +23,7 @@ def make_plugin_or_core():
     use_core_openvino = False
     try:
         major, minor, reference = [int(x) for x in version.split('.')]
-        if major >= 2 and minor >= 1 and reference >= 37988:
```

Comment:
@benhoff , why the condition isn't important anymore?

Reply:
The `reference` condition will continue to change. In order to test this, I had to compile dldt on my machine. My `reference` was completely different. 

My concern was that when they roll the release to a 3.1.XXXXX, the reference number might be smaller than `37988`, causing this line to malfunction.

This might also happen for a 2.2.XXXXX